site/es/blog/new-in-devtools-100/index.md

```diff
@@ -28,33 +27,50 @@ _Gracias [Miguel Ángel](https://midu.dev) por la traducción y [Carlos Caballer
 <!-- end: translation instructions -->
```

Comment:
Please update the `whats-new.md` file too!

Reply:
Done!